.github/config.yml

```diff
@@ -0,0 +1,14 @@
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here at xarray! Be sure to follow the issue template!
+  ![coding gif](https://media.giphy.com/media/wv1RNuvWMjQ10bzExO/giphy.gif)
```

Comment:
```suggestion
  Thanks for opening your first issue here at xarray! Be sure to follow the issue template!
  ![coding gif](https://media.giphy.com/media/wv1RNuvWMjQ10bzExO/giphy.gif)

  If you have an idea for a solution, we would really welcome a Pull Request with proposed changes.
  See the [Contributing Guide](https://docs.xarray.dev/en/latest/contributing.html) for more.

  It may take us a while to respond here, but we really value your contribution. Contributors like you
  help make xarray better. Thank you!
```

How about something like this ? @nishtha981 are there other things that would've been useful to know when you opened your first issue / PR?

Reply:
Sure!
I'll remove them and commit these new changes!